docs/generate_examples.py

```diff
@@ -102,6 +102,7 @@ def main():
     BASE_IMAGEOUTDOOR_URL8: str = (
         "https://github.com/kornia/data/raw/main/kornia_banner_pixie.png"  # Response functions
     )
+    MASK_IMAGE_URL2: str = "https://files.jansellner.net/simba_mask.png"
```

Comment:
TODO: change link

Reply:
You can open a PR on https://github.com/kornia/data/